ops/charms/node_base.py

```diff
@@ -104,7 +124,7 @@ def set_label(self, label: str, value: str) -> None:
         cmd = self._kubectl("label node {0} {1}={2} --overwrite")
         cmd = cmd.format(self.charm.get_node_name(), label, value)
         retry_msg = "Failed to apply label {0}={1}. Will retry.".format(label, value)
-        LabelMaker._retried_call(cmd.split(), retry_msg)
+        self._retried_call(cmd.split(), retry_msg)
```

Comment:
What if we use `shlex.split` instead of `str.split`? It's better suited for handling this shell-like expressions

Reply:
sure, let's go for it